### ISSUE TITLE
[BUG] Export layer to Mesh generates an STL file close to 0 bytes

### ISSUE THREAD

**Issue by ghost** [sn4k3/UVtools#560]
Hallo, as declared the "Export layer to Mesh" of a pwma file generates a STL object of few bytes, unreadable. 

Below there is a ZIP file of both

[adapter.zip](https://github.com/sn4k3/UVtools/files/9614657/adapter.zip)

**Comment by sn4k3** [comment]
Interesting, if you move that object to the center, it works on stl

**Comment by tecchese** [comment]
yes, placing it at center it works...it is a weird behavior anyway...

**Comment by sn4k3** [comment]
Fixed already
